src/Database/Database.php

```diff
@@ -152,6 +152,10 @@ function($value) {
                     return $value;
                 }
 
+                if($value === '{}') {
+                    return new \stdClass();
+                }
+
                 $value = json_decode($value, true);
```

Comment:
```suggestion
                $value = json_decode($value);
```

This might be the issue